### ISSUE TITLE
Names disspear in edit mode and contact details view

### ISSUE THREAD

**Issue by ph00lt0** [SimpleMobileTools/Simple-Contacts#862]
Since (one of) the last updates names are not shown when opening a contact and when editing the contact they disappear completely. Quite a critical bug I would say.

**Comment by ph00lt0** [comment]
As you can see the name row in the details does not appear. 
Interrestingly the name also doesn't start with 'I' but this is shown in the contact image. 

In the list view the name and image appear correctly as well as in my default contacts app, which I had to enable again. 

![signal-2022-08-08-171452](https://user-images.githubusercontent.com/15004290/183464428-8853ac16-0d7a-41fd-abc6-06c05d42be23.jpeg)
![signal-2022-08-08-171524](https://user-images.githubusercontent.com/15004290/183464442-5e501f34-290b-4aed-b23c-ba4e55c467fe.jpeg)

**Comment by ph00lt0** [comment]
Just tested a bit further it seems this only happens when the contact has a company field.

**Comment by tibbi** [comment]
well, why would the contacts name show if it has no name? It is a company

**Comment by ph00lt0** [comment]
I don't think you understand it. It isn't a company. It is a contact who works at a company?!

**Comment by ph00lt0** [comment]
@tibbi sorry but this isn't the case. Not sure why you are closing this bug report.

**Comment by ph00lt0** [comment]
I store my contacts solemnly in my DAV5x server and any other contacts app shows this without any issue. I can see on the server that the details are correct

**Comment by tibbi** [comment]
I dont see any issue on the screenshots

**Comment by ph00lt0** [comment]
The fields are empty? the name does not appear? how is that not an issue?

**Comment by tibbi** [comment]
names are empty because it is an organization..

**Comment by ph00lt0** [comment]
no it isn't

**Comment by ph00lt0** [comment]
![signal-2022-08-08-215537](https://user-images.githubusercontent.com/15004290/183512945-aea91137-f915-4c35-832a-f99dbaed24cd.jpeg)
![signal-2022-08-08-215513](https://user-images.githubusercontent.com/15004290/183512992-04f32c7c-f170-49de-be95-cac5d1ddc204.jpeg)

I hope you see the difference and the problem now.
